### ISSUE TITLE
Special Modes - Print Sequence Script/Mod to Skip or Next printing object

### ISSUE THREAD

**Issue by MRDavidas** [Ultimaker/Cura#7315]
**Is your feature request related to a problem? Please describe.**
I am printing these 25 pieces(off 1000) one by one. using Special Modes - Print Sequence .
![Capture](https://user-images.githubusercontent.com/44553118/77169651-c7ad4b00-6ab1-11ea-8cb4-d5fddf9a8fb1.JPG) 
**Describe the solution you'd like**
is it possible somehow to make skip model if it fails to print and jump to the next one? 
**Describe alternatives you've considered**
Thinking to have add script in the G-code Where you can skip/jump to next layer of model to start the new print.
**Affected users and/or printers**
it will allow saving your printing time, used filament. If a model fails you can easily skip not to start over.
**Additional context**
(Add any other context or screenshots about the feature request here.)

**Comment by rburema** [comment]
>  if it fails to print

We don't get any information back from the printer... slicing is not a live process, we slice once, and then send the gcode to the printer.

> add script in the G-code Where you can skip/jump to next layer of model to start the new print

As far as I'm aware, there's no standard gcode for 'pause on failure and let the user decide to skip an arbitrary amount of instructions' or similar.

Features like these will need to be added on the printer side of things _first_, before we can even think about adding them to Cura.

**Comment by MRDavidas** [comment]
Hi , 

This is your decision to close. 
> 
> if it fails to print
> 
> We don't get any information back from the printer... slicing is not a live process, we slice once, and then send the gcode to the printer.
> 
You not need to back information from printer. You can have your eyes  to see when it fails.
> add script in the G-code Where you can skip/jump to next layer of model to start the new print
> 
> As far as I'm aware, there's no standard gcode for 'pause on failure and let the user decide to skip an arbitrary amount of instructions' or similar.
> 
For example octoprint has this feature. You can close/cancel object whaling is printing. 
You need a script in the G-code when you pushed button goes to next layer. 
I can even do it now. But in the 1000 lines it is very hard to do manually.

> Features like these will need to be added on the printer side of things first, before we can even think about adding them to Cura.
> 
Your product is your decision. I always can use octoprint and less Cura . 
Thanks for your time.

**Comment by MRDavidas** [comment]
![B8B92D57-16FB-45CE-97F5-0D549C9879E8](https://user-images.githubusercontent.com/44553118/77225118-3a76fe80-6b64-11ea-9822-d074094ab0bc.png)

**Comment by MRDavidas** [comment]
https://github.com/MarlinFirmware/Marlin/pull/15590

**Comment by fieldOfView** [comment]
You could try the postprocessing script that is linked from that issue:
https://github.com/LinFor/Cura/blob/e-bot/plugins/PostProcessingPlugin/scripts/MarlinObjectsMarking.py

**Comment by Ghostkeeper** [comment]
Duplicate of https://github.com/ultimaker/cura/issues/7014

**Comment by rburema** [comment]
@MRDavidas I'm sorry, I misunderstood.
